<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPage</class>
 <widget class="QWizardPage" name="WizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Loading options</string>
  </property>
  <property name="subTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="loadInXMLRadio">
     <property name="text">
      <string>Load in XML &amp;mode</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="loadInRelationalRadio">
     <property name="text">
      <string>Load in relational &amp;mode (GMLAS)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="donwloadCheck">
       <property name="text">
        <string>Download to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputPathLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The file downloaded will be saved in this location. If not set, a temporary file is created.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="placeholderText">
        <string>Create temporary file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="outputPathButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downloadButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>donwloadCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>downloadButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>donwloadCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>outputPathButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>donwloadCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>outputPathLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
